disabled/epochNotifier.go

```diff
@@ -10,6 +13,13 @@ type EpochNotifier struct {
 func (en *EpochNotifier) RegisterNotifyHandler(handler vmcommon.EpochSubscriberHandler) {
 }
 
+func (en *EpochNotifier) CurrentEpoch() uint32 {
+	return 0
+}
+
+func (en *EpochNotifier) CheckEpoch(header data.HeaderHandler) {
```

Comment:
can you use _ for unused parameters? 
instead of header here, and handler (RegisterNotifyHandler)

Reply:
done, also for `RegisterNotifyHandler()`